plugins/module_utils/ndi.py

```diff
@@ -272,9 +272,16 @@ def query_unhealthy_resources(self, ig_name, site_name, compliance_epoch_id):
         return result
 
     def query_pcvs(self, ig_name):
-        pcvs_path = "{0}/{1}/prechangeAnalysis?$sort=-analysisSubmissionTime".format(self.config_ig_path, ig_name)
-        obj = self.nd.query_obj(pcvs_path, prefix=self.prefix)
-        return obj["value"]["data"]
+        page = 0
+        pcvs_data = []
+        has_more_data = True
+        while has_more_data:
+            pcvs_path = "{0}/{1}/prechangeAnalysis?$size=50&$page={2}&$sort=-analysisSubmissionTime".format(self.config_ig_path, ig_name, page)
```

Comment:
I'm just curious, does this filter query ?$size=50 mean 50 and greater in NDI?

Reply:
@shrsr the size parameter determines the max number of items that should be returned in the request. It already defaults to 50 when not specified. I wanted to be explicit in case the API changes so we maintain a consistent behaviour.